samples/interactive-tutorials/src/main/java/events/PurgeUserEvent.java

```diff
@@ -58,7 +58,7 @@ public static void callPurgeUserEvents(String defaultCatalog, String visitorId)
     try (UserEventServiceClient userEventServiceClient = UserEventServiceClient.create()) {
       PurgeUserEventsRequest purgeUserEventsRequest =
           PurgeUserEventsRequest.newBuilder()
-              // TO CHECK ERROR HANDLING SET INVALID FILTER HERE:
+              // To check error handling set invalid filter here:
               .setFilter(String.format("visitorId=\"%s\"", visitorId))
               .setParent(defaultCatalog)
               .setForce(true)
```

Comment:
Do we want to add a comment mentioning why are we doing a  `setForce(true)`

Reply:
Added comment with the explanation.